components/lib/password/Password.js

```diff
@@ -272,16 +272,22 @@ export const Password = React.memo(
             ZIndexUtils.clear(overlayRef.current);
         });
 
+        const onToggleMaskKeyDown = (e) => {
+            return e.key === 'Enter' ? toggleMask() : '';
```

Comment:
Do you need the `return` here?
```suggestion
            e.key === 'Enter' && toggleMask();
```